R/logical.R

```diff
@@ -9,6 +9,9 @@
 #' within a messy date range or set.
 #' `is_similar()` tests whether two dates contain similar components.
 #' This can be useful for identifying dates that may be typos of one another.
+#' `is_uncertain()` tests whether a date is uncertain.
+#' Uncertain dates can contain markers that they are approximate (i.e. ~),
```

Comment:
I think the repetition is confusing, should we just use 'uncertain' to refer to all these dates and 'unreliable' to refer to dates that are annotated with '?' ? Also are we using % anywhere? I think it's not in the README

Reply:
Thank you for the comment Jael! You are right, this may be confusing. I will change the name of this logical tests is_precise() instead and modify it to test whether date is precise or not.